src/main/scala/com/dataartisans/flinktraining/exercises/dataset_scala/tf_idf/MailTFIDF.scala

```diff
@@ -1,11 +1,17 @@
-/*
+package org.apache.flink
+
+/**
  * Copyright 2015 data Artisans GmbH
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please do not change the license header. 
This code is not part of Apache Flink and does not belong to the ASF.